src/index.html

```diff
@@ -7,6 +7,385 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__icon">
+        <img src="images/logo.png" alt="moyo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link--active is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover">ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="content content--margins">
```

Comment:
use mix classes for outer spacing
https://www.youtube.com/watch?v=JOJ-mENrg_s

```suggestion
    <main class="container">
      <div class="content container__content">
```